db/migrate/20231117182638_fill_approve_unless_allowed_groups_based_on_deprecated_settings.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+class FillApproveUnlessAllowedGroupsBasedOnDeprecatedSettings < ActiveRecord::Migration[7.0]
```

Comment:
We usually have an empty line between these two.

Reply:
oh ya, I had rubocop add this line in, I wonder if we can configure that somehow?

Reply:
The setting is: `EmptyLineAfterMagicComment`.

So in our config we could add:

```yml
Layout/EmptyLineAfterMagicComment:
  Enabled: true
```